WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/contacts/ContactsFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.example.whatsapp.ui.fragments.contacts
+
+import android.os.Bundle
+import android.view.*
+import androidx.fragment.app.Fragment
+import androidx.activity.addCallback
+import androidx.appcompat.app.AppCompatActivity
+import androidx.databinding.DataBindingUtil.setContentView
+import androidx.navigation.NavController
+import androidx.navigation.ui.navigateUp
+import androidx.navigation.ui.onNavDestinationSelected
+import com.example.whatsapp.R
+
+
+class ContactsFragment : Fragment() {
+    private lateinit var navController: NavController
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        setHasOptionsMenu(true)
+        super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.fragment_contacts, container, false)
```

Comment:
set up view binding and also inflate the Fragment inside of the Fragment() class on line 15.

Fragment(R.layout.fragment_contacts)